packages/reactivity/src/dep.ts

```
@@ -3,10 +3,14 @@ import type { ComputedRefImpl } from './computed'
 import { type TrackOpTypes, TriggerOpTypes } from './constants'
 import {
   type DebuggerEventExtraInfo,
+  type DebugLineageNode,
```

ESLint

[error] Member 'DebugLineageNode' of the import declaration should be sorted alphabetically.

(sort-imports)